pkg/api/convert.go

```diff
@@ -17,6 +20,7 @@ import (
 // string                <-- string
 // time.Duration         <-- string
 // time.Time             <-- string (only in RFC3339 format)
+// IntOrStr              <-- string or int64
```

Comment:
```suggestion
// IntOrStr              <-- string or int32
```

Reply:
Does goja support `int64`s? I though all JS numbers were spec'd as 64b IEEE floats, but I'm not sure how goja does it here.